docs/configuration.md

```
@@ -0,0 +1,67 @@
+---
+icon: lucide/settings
+---
+
+# Configuration
+
+## Environment variables
+
+All settings come from [StareSettings][stare.settings.StareSettings], which
```

markdownlint-cli2 (0.22.0)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "stare.settings.staresettings"

(MD052, reference-links-images)